en/docs/scheduled-tasks.md

```diff
@@ -1,23 +1,73 @@
 # Scheduled Tasks
 
-A scheduled task performs a routine task at scheduled time intervals.
+Learn how to execute routine tasks at specified time intervals via scheduled tasks.
```

Comment:
Learn how to use scheduled tasks to execute routing tasks at specified time intervals? -> Wouldn't this be clearer?

Reply:
Changed as suggested.